File: app/src/main/java/com/shindra/Schedule/ScheduleActivity.java
package com.shindra.Schedule;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentTransaction;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;

import com.shindra.R;

public class ScheduleActivity extends AppCompatActivity {

    // Attributes
    private static final String TAG = "ScheduleActivity";
    private Fragment mFragment;

    // Methods
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        Log.i(TAG, "onCreate");
        super.onCreate(savedInstanceState);
        setContentView(R.layout.av_schedule);

        // Retrieve intent data
        Intent intent = getIntent();
        String tramLineLetter = intent.getStringExtra("tramLineLetter");

        // Create, pass args & call fragment
        mFragment = ScheduleFragment.onInstance(tramLineLetter);
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.add(R.id.av_schedule_fragmentHolder, mFragment).commit();
    }
}

File: app/src/main/java/com/shindra/schedule/ScheduleActivity.kt
package com.shindra.schedule

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.shindra.R

class ScheduleActivity : AppCompatActivity() {

    // Attributes
    private lateinit var tramLineKey: String
    private lateinit var mFragment: ScheduleFragment
    private var mTramLineLetter: String? = null

    // Methods
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.av_schedule)
        tramLineKey = getString(R.string.TramLineLetterGen)

        // Retrieve intent data
        val intent = intent
        mTramLineLetter = intent?.getStringExtra(tramLineKey)

        // Create, pass args & call fragment
        mFragment = ScheduleFragment.onInstance(mTramLineLetter, tramLineKey)
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.add(R.id.av_schedule_fragmentHolder, mFragment).commit()
    }
}